syntax = "proto3";

option go_package = "gitlab.com/timeterm/timeterm/proto/go;timetermpb";

package timeterm_proto.messages;

message DisownTokenMessage {
    string device_id = 1;
    string token_hash = 2;
    string token_hash_alg = 3;
}

message RetrieveNewTokenMessage {
    string device_id = 1;
    string current_token_hash = 2;
    string current_token_hash_alg = 3;
}

